tools/model-compiler/src/main/java/io/spine/tools/compiler/validation/VBuilderGenerator.java

```diff
@@ -73,10 +74,12 @@ public VBuilderGenerator(File protoSrcDir, File targetDir, Indent indent) {
     public void process(FileSet files) {
         FileSet fileSet = moduleFiles(files);
         ImmutableCollection<MessageType> messageTypes = TypeSet.onlyMessages(fileSet);
+        Predicate<MessageType> notRejection =
+                ((Predicate<MessageType>) MessageType::isRejection).negate();
         ImmutableList<MessageType> customTypes =
                 messageTypes.stream()
                             .filter(MessageType::isCustom)
-                            .filter(MessageType::isNotRejection)
+                            .filter(notRejection)
```

Comment:
We can use `Predicates.not(MessageType::isRejection)` here, with a static import it looks cleaner to me: `not(MessageType::isRejection)`.